.github/workflows/release-image.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generate Release Image
+
+on:
+  pull_request:
+    paths:
+      - 'CHANGELOG.md'
```

Comment:
I think this is redundant with `'**/CHANGELOG.md'` that's already in the line below.

Reply:
Great point! Thank you!